Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -43,14 +43,18 @@ final class LayerTests: XCTestCase {
             let optimizer = SGD(for: model)
             let x = Tensor<Float>([[0, 0], [0, 1], [1, 0], [1, 1]])
             let y = Tensor<Float>([0, 1, 1, 0])
-            let initialLoss = meanSquaredError(predicted: model(x).squeezingShape(at: 1), expected: y)
-            for _ in 0..<10 {
-                let 𝛁model = model.gradient { model -> Tensor<Float> in
-                    meanSquaredError(predicted: model(x).squeezingShape(at: 1), expected: y)
+            let initialLoss = meanSquaredError(predicted: model(x).squeezingShape(at: 1),
```

Comment:
How about this?
```swift
let initialLoss = meanSquaredError(
    predicted: model(x).squeezingShape(at: 1),
    expected: y)
```
I guess we should decide on a style and document it somewhere.

Reply:
Done. I lean towards Google Swift Style Guide's recommendations although I don't always remember to follow it.